examples/transactions/examples/encode_and_decode_raw_transaction.rs

```diff
@@ -0,0 +1,102 @@
+//! Example of encoding and decoding raw transactions.
+
+use alloy::primitives::keccak256;
+use alloy::primitives::private::alloy_rlp::Decodable;
+use alloy::primitives::private::alloy_rlp::Encodable;
+use alloy::primitives::FixedBytes;
+use alloy::primitives::TxKind;
+use alloy::primitives::U256;
+use alloy::providers::WalletProvider;
+use alloy::{
+    consensus::TxEnvelope,
+    consensus::{SignableTransaction, TxEip1559},
+    hex,
+    network::{EthereumWallet, TransactionBuilder},
+    primitives::Address,
+    providers::{Provider, ProviderBuilder},
+    rpc::types::TransactionRequest,
+};
+use eyre::Result;
+
+fn build_unsigned_tx(chain_id: u64, to_address: Address) -> TxEip1559 {
+    let mut tx = TxEip1559::default();
+    tx.chain_id = chain_id;
+    tx.nonce = 0;
+    tx.gas_limit = 21_000;
+    tx.max_fee_per_gas = 20_000_000_000;
+    tx.max_priority_fee_per_gas = 1_000_000_000;
+    tx.to = TxKind::Call(to_address); // Change this to `TxKind::Create` if you'd like to deploy a contract instead
+    tx.value = U256::from(100);
+
+    tx
+}
```

Comment:
We can use the builder properties of TransactionRequest here

https://github.com/alloy-rs/examples/blob/5a6776f2400312ee6beb7ee108d9407cd889c078/examples/transactions/examples/send_eip1559_transaction.rs#L22-L31

Reply:
I didn't find the method of RLP-Encoding in the TransactionRequest struct. 

So Thats why I use TxEip1559. but I changed the coding style

```rust
fn build_unsigned_tx(chain_id: u64, to_address: Address) -> TxEip1559 {
    TxEip1559 {
        chain_id,
        nonce: 0,
        gas_limit: 21_000,
        max_fee_per_gas: 20_000_000_000,
        max_priority_fee_per_gas: 1_000_000_000,
        to: TxKind::Call(to_address), // Change this to `TxKind::Create` if you'd like to deploy a contract instead
        value: U256::from(100),
        ..Default::default()
    }
}
```

Reply:
> I couldn't find the RLP-encoding-method in the TransactionRequest struct.

there is no RLP-encoding method becasue `TransactionRequest` cannot be RLP encoded. you should use [`build`](https://docs.rs/alloy/latest/alloy/network/trait.TransactionBuilder.html#tymethod.build) to construct and sign a transaction, converting `TransactionRequest` into `TransactionEnvelope`